src/main/resources/db/migration/all/20240607134906__create_legacy_placement_applications_automatic_records.sql

```diff
@@ -0,0 +1,12 @@
+-- create placementApplicationsAutomatic records for existing applications with arrival date
```

Comment:
very nice!

can we please replace the the word 'create' in the filename with 'backfill' as that's the common term we use to describe these operations. I think we can then remove the comment in the file too.